js/plugins/drupalurl/plugin.js

```diff
@@ -27,8 +27,8 @@
 
       // Generic command for adding/editing entities of all types.
       editor.addCommand('editdrupalurl', {
-        allowedContent: 'drupal-url[*]',
-        requiredContent: 'drupal-url[*]',
+        allowedContent: 'drupal-url[data-embed-url,data-url-provider]',
+        requiredContent: 'drupal-url[data-embed-url,data-url-provider]',
```

Comment:
data-url-provider is not a required attribute to render content, it's just additional data.